contracts/Recruitment.sol

```diff
@@ -18,6 +18,26 @@ contract Recruitment {
     owner = msg.sender;
   }
 
+  struct ReferralScore {
+        uint256 score;  //Score given by the hiring company to the candidate 
+        address senderAddress; // Wallet address of the hiring company
+  }
+  event ReferralScoreSubmitted(address senderAddress, address referrerWallet, string referrerEmail, uint256 score);
+  mapping(address => ReferralScore[]) public referralScores;
+
+  function submitReferralScore(uint256 score,address senderAddress,address referrerWallet,string memory referrerEmail) public returns (bytes32) {
```

Comment:
Sorry the user story was not groomed enough. Your code is good but the email bullet was not part of input params. It means an email should be sent after this transaction happens (my bad). Can you please remove this param and logic related to email here. We do not need it in this transaction. Thanks :)

Also instead of passing the senderAddress (hiring company) as param and since this is a write transaction can use the built in "sender.address" ? thanks a lot :)

Reply:
Sure done with the changes